poet.go

```diff
@@ -81,18 +87,18 @@ func poetMain() error {
 	if cfg.CPUProfile != "" {
 		f, err := os.Create(cfg.CPUProfile)
 		if err != nil {
-			log.Error("Could not create CPU profile: ", err)
+			logger.With(zap.Error(err)).Error("could not create CPU profile")
 		}
 		defer f.Close()
 		if err := pprof.StartCPUProfile(f); err != nil {
-			log.Error("Could not start CPU profile: ", err)
+			logger.With(zap.Error(err)).Error("could not start CPU profile")
```

Comment:
i find the inconsistency of usage confusing/distracting.
can you consistently use logger.Error("", zap.Error(err)) instead?

Reply:
Fixed